server/presenters/user.ts

```diff
@@ -18,6 +19,7 @@ type UserPresentation = {
   isViewer: boolean;
   email?: string | null;
   language?: string;
+  preferences?: UserPreferences;
```

Comment:
> Do we want that?
Yes, I think so.

We will need to validate the keys on the server but otherwise it should be settable from the client side

Reply:
> We will need to validate the keys on the server but otherwise it should be settable from the client side

Sorry, this is in the presentation layer. I assumed we want to _restrict the visibility_ for user preferences similar to the way we've restricted `email` and `language`. Regarding validation and update, we're already doing that via `users.update`. The latter has nothing to do with former, right? Both are disjoint flows? Let me know if I misunderstood something :smile:

Reply:
You're correct, preferences should only be sent to the user they related to. Sorry for misdirection.

Reply:
Thanks! I myself realized only after pushing the suggested changes. Will revert now :laughing: